app/frontends/workflow-ui.ts

```diff
@@ -295,7 +295,7 @@ export async function getJob(
   try {
     const isAdmin = await isAdminUser(req);
     const job = await getJobIfAllowed(jobID, req.user, isAdmin, req.accessToken, true);
-    const { page, limit } = getPagingParams(req, 1000);
+    const { page, limit } = getPagingParams(req, 100);
```

Comment:
I don't think we have to make these configuration parameters, but rather than using numbers directly can you define constants for them and then use those constants - e.g. `const defaultWorkItemPageSize = 100` just as a documentation thing.

Reply:
Added the const.

Reply:
Also added "...changed your value to 2000." by passing in useMaxWhenExceeded=true to getPagingParams.